JSQMessagesViewController/Model/JSQMessagesViewAccessoryButtonConfiguration.h

```diff
@@ -0,0 +1,63 @@
+//
+//  Created by Andrey Dubrovskiy
+//  https://github.com/adubr
+//
+//  Documentation
+//  http://cocoadocs.org/docsets/JSQMessagesViewController
+//
+//
+//  GitHub
+//  https://github.com/jessesquires/JSQMessagesViewController
+//
+//
+//  License
+//  Copyright (c) 2014 Jesse Squires
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
```

Comment:
for all new files, let's keep the comment headers consistent with the rest of the project 😄